tests/next_tests/unit_tests/conftest.py

```diff
@@ -82,7 +82,7 @@ def program_processor(request) -> tuple[ppi.ProgramProcessor, bool]:
     for marker, skip_mark, msg in next_tests.definitions.BACKEND_SKIP_TEST_MATRIX.get(
         processor_id, []
     ):
-        if request.node.get_closest_marker(marker):
+        if marker == next_tests.definitions.ALL or request.node.get_closest_marker(marker):
```

Comment:
I would prefer a different approach. I would change line 39 to:
`if dace_iterator and False:`